kbase-extension/static/kbase/js/widgets/appWidgets2/input/textInput.js

```diff
@@ -97,7 +103,9 @@ define([
                     cancelTouched();
                     autoChangeTimer = window.setTimeout(() => {
                         autoChangeTimer = null;
-                        e.target.dispatchEvent(new Event('change'));
+                        if (e.target) {
+                            e.target.dispatchEvent(new Event('change'));
+                        }
```

Comment:
This was causing a test to fail. I'm not sure if the test runner doesn't always include the full event object when dispatching an event manually or if something else strange is going on. But wrapping this in an `if` fixes it.